src/pages/home/report/ReportActionCompose.js

```diff
@@ -90,6 +91,9 @@ class ReportActionCompose extends React.Component {
      * @param {String} newComment
      */
     updateComment(newComment) {
+        this.setState({
+            isCommentEmpty: newComment
```

Comment:
This works, but we should use a boolean instead of setting the comment itself to state. See above comment for suggestion on how to do this.

Reply:
done